cmd/main.go

```diff
@@ -93,8 +93,9 @@ type Config struct {
 		MinTarget          int `yaml:"minTarget"`
 		MaxTarget          int `yaml:"minTarget"`
 	} `yaml:"cpuUtilizationBasedRecommender"`
-	MetricIngestionTime float64 `yaml:"metricIngestionTime"`
-	MetricProbeTime     float64 `yaml:"metricProbeTime"`
+	MetricIngestionTime      float64 `yaml:"metricIngestionTime"`
+	MetricProbeTime          float64 `yaml:"metricProbeTime"`
+	EventCalendarAPIEndpoint string  `yaml:"eventCalendarAPIEndpoint"`
```

Comment:
Also make this entire outlier detection based on event calendar behind a feature flag and disable by default as this feature is not applicable to the oss version of this component.